source/_integrations/climate.markdown

```diff
@@ -216,6 +216,30 @@ automation:
         swing_mode: 1
 ```
 
+### Action `climate.set_swing_horizontal_mode`
+
+Set horizontal swing operation mode for climate device
+
+| Data attribute          | Optional | Description                                                                                                                       |
+| ----------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| `entity_id`             | yes      | String or list of strings that define the entity ID(s) of climate device(s) to control. To target all climate devices, use `all`. |
+| `swing_horizontal_mode` | no       | New value of horizontal swing mode.                                                                                               |
```

Comment:
```suggestion
| `swing_horizontal_mode` | no       | New value of horizontal swing mode. Possible values are `on` or `off`.                                                                                              |
```
Please correct me if I got this wrong. In any case, I think it would be worth mentioning the possible values.

Reply:
It's not limited to those two values as every integration implementing this will for sure add multiple own states which are integration specific.
So I don't think we should be specific here with possible values.